tests/test_requirements.py

```diff
@@ -81,10 +81,10 @@ def test_requirement_specifiers_convention():
         "kfp": {"~=1.8.0"},
         "botocore": {">=1.20.106,<1.20.107"},
         "aiobotocore": {"~=1.4.0"},
-        "aioitertools": {"<0.9"},
+        "aioitertools": {""},
```

Comment:
Not sure whether it fails the test, but pretty sure this line should go.

Reply:
removed